tenzir/integration/tests/http.bats

```diff
@@ -1,39 +1,54 @@
 : "${BATS_TEST_TIMEOUT:=10}"
 
-export TENZIR_PLUGINS="fluent-bit"
-
-wait_for_tcp() {
-  port=$1
-  timeout $BATS_TEST_TIMEOUT bash -c "until echo > /dev/tcp/127.0.0.1/$port; do sleep 0.2; done"
-}
-
 setup() {
   bats_load_library bats-support
   bats_load_library bats-assert
   bats_load_library bats-tenzir
 }
 
-@test "saver" {
-  # TODO: Write convenience function to get current tenzir config
-  # TODO: Why is this test not in the fluent-bit plugin?
-  setup_node_with_default_config
-  has_fluentbit=$(tenzir 'show plugins | where name == "fluent-bit"')
-  teardown_node
+load() {
+  if ! command -v python; then
+    skip "python executable must be in PATH"
+  fi
+  check --bg server python "${MISCDIR}/scripts/webserver.py"
+  check tenzir "$@"
+  wait_all ${server[@]}
+}
 
-  if [ -z $has_fluentbit ]; then
-    skip "built without fluent-bit support"
+save() {
+  if ! command -v python; then
+    skip "python executable must be in PATH"
   fi
+  check --bg server python "${MISCDIR}/scripts/webserver.py"
+  tenzir "$@"
+  wait_all ${server[@]}
+}
+
+@test "load HTTP POST" {
+  load 'load http://localhost:8888 foo=42'
```

Comment:
These tests do not correctly run in parallel because they all use the same port. 8888 is also a rather commonly used port, and because CI runners may be shared this is likely to cause sporadic CI failures in CI.

Other than that this is all good—just please use a port that's less commonly used and use different ports for each test.